src/plugins/i18n.ts

```diff
@@ -13,6 +13,9 @@ export default createI18n({
       }
       const teen = choice > 10 && choice < 20
       const endsWithOne = choice % 10 === 1
+      if (choicesLength == 2) {
+        return choice === 1 ? 0 : 1
```

Comment:
I would include 0 and negative values in other languages. Could that be working with ru grammar?

```suggestion
        return choice <= 1 ? 0 : 1
```

Reply:
Can you give an example?

Reply:
Well for example in French and English, plural is applied when the associated numeric value is greater than 1, so all negative numbers, 0, 1, 0.5, ... are singular and 2, 1.1, 10000 are plural

Reply:
After checking, this only applies in French. In English the 0 value is also plural.